CHANGELOG.md

```diff
@@ -61,6 +61,7 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Added `MessageListView::addOnScrollListener` and `MessageListView::removeOnScrollListener` methods to the `MessageListView` to listen to scroll events of the inner message list.
```

Comment:
Shouldn't we also add an example of how to use these APIs to the docs?

Reply:
Seems like a request from a customer for a functionality which will be not very popular among other clients. What do you think, should we create an example anyway?